etherman/etherman.go

```diff
@@ -198,7 +198,7 @@ func (etherMan *Client) EstimateGasSequenceBatches(sequences []ethmanTypes.Seque
 	if err != nil {
 		return 0, err
 	}
-	return tx.Cost().Uint64(), nil
+	return tx.Gas(), nil
```

Comment:
Returned back, bcs it was impossible to send a tx with tx.Cost instead of tx.Gas. A request was failing with "exceeds block gas limit" even after 10 times increase